### PR TITLE
RCT/YJ./branch_coverage_is_baseline_system

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
This PR addresses achieving 100% branch coverage for all the ```is_baseline_system_X``` functions.

### CODE REVIEW

File: rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/test_is_baseline_system_10.py [hunk 1]
```diff
@@ -53,24 +108,36 @@
...
             "System 10",
-            ["Air Terminal"],
+            ["Air Terminal 1"],
             ["Thermal Zone 1"],
         )
         == HVAC_SYS.SYS_10
     )
 
 
+def test__is_baseline_system_10__second_logic_true():
```
Comment (weilixu) [inline rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/test_is_baseline_system_10.py:129]: Could you add comments to distinguish the first logic and second logic?

File: rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/test_is_baseline_system_11_1.py [hunk 2]
```diff
@@ -312,3 +444,27 @@ def test__is_baseline_system_11_1C__test_json_true():
...
+            SYS_11_1_TEST_UNMATCHED_RMD["ruleset_model_instances"][0],
+            "System Type Unmatched 1",
+            ["Air Terminal 1"],
+            ["Thermal Zone 1"],
+        )
+        == HVAC_SYS.UNMATCHED
+    )
+
+
+def test__is_baseline_system_unmatched2__true():
```
Comment (weilixu) [inline rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/test_is_baseline_system_11_1.py:461]: Similarly, give a short description of the test logic here will be great.

**Review by weilixu — APPROVED** [review]
Approved with some comments

File: rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/test_is_baseline_system_10.py (hunk 1 above)
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/test_is_baseline_system_10.py:129]: Good point! Sure. I just added the comment about the first and second logic.

File: rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/test_is_baseline_system_11_1.py (hunk 2 above)
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/test_is_baseline_system_11_1.py:461]: Good point! I just added the description.

File: .coveragerc [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [run]
 branch = True
-omit = **/test_*, **/*_test*.*, **/__init__.py, **/rules/**
+omit = **/test_*, **/*_test*.*, **/__init__.py, **/section/**
```
Comment (charliepnnl) [inline .coveragerc:3]: I think you need `**/section*/**`.

File: rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/is_baseline_system_11_1.py
Comment (charliepnnl) [inline rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/is_baseline_system_11_1.py:1]: I like the removal of `_str`.

File: rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/is_baseline_system_7.py [hunk 4]
```diff
@@ -94,7 +94,7 @@ def is_baseline_system_7(rmi_b, hvac_b_id, terminal_unit_id_list, zone_id_list):
         and are_all_terminal_fans_null(rmi_b, terminal_unit_id_list)
         and are_all_terminal_types_VAV(rmi_b, terminal_unit_id_list)
     )
-
+    stop = 1
```
Comment (charliepnnl) [inline rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/is_baseline_system_7.py:97]: What is the purpose of this line of code? `stop = 1`

**Review by charliepnnl — APPROVED** [review]
Looks good

File: rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/is_baseline_system_7.py (hunk 4 above)
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/is_baseline_system_7.py:97]: My apologies. This was just my stopping point for debugging. This was fixed.

**Pull request merged**